contracts/core/Controller.sol

```diff
@@ -169,21 +169,23 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
     );
     /// @notice emits an event when a vault is settled
     event VaultSettled(
-        address indexed AccountOwner,
-        address indexed to,
-        address indexed otoken,
+        address indexed accountOwner,
+        address indexed oTokenAddress,
+        address payoutRecipient,
```

Comment:
Can we keep this to `to` to match the action argument ? IMO better to keep the same naming whenever we can.

Reply:
Same renaming suggestion here. `payoutRecipient` is much more explicit than `to` :joy:.

Reply:
I agree, but I guess it require some changes in Controller that is not worth it IMO.